glean-core/ios/Glean/Utils/Utils.swift

```diff
@@ -141,3 +141,36 @@ func timestampNanos() -> UInt64 {
     let nanos = currentTime * UInt64(info.numer) / UInt64(info.denom)
     return nanos
 }
+
+/// Gets a gecko-compatible locale string (e.g. "es-ES")
+///
+/// - returns: a locale string that supports custom injected locale/languages.
+func getLocaleTag() -> String {
+    if NSLocale.current.languageCode == nil {
+        return "??"
+    } else {
+        if NSLocale.current.regionCode == nil {
+            return NSLocale.current.languageCode!
+        } else {
+            return "\(NSLocale.current.languageCode!)-\(NSLocale.current.regionCode!)"
+        }
+    }
+}
```

Comment:
Just to confirm -- I assume iOS doesn't have any of the out-of-date locale stuff that we have to workaround [here](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/src/main/java/mozilla/components/service/glean/utils/LocaleUtils.kt#L38)?

Reply:
I'm pretty sure Apple has this right, but good point

Reply:
We're fine.